.vscode/settings.json

```diff
@@ -16,8 +16,8 @@
   "dart.onlyAnalyzeProjectsWithOpenFiles": true,
   "files.insertFinalNewline": true,
   "editor.codeActionsOnSave": {
-    "source.fixAll": true,
-    "source.organizeImports": true
+    "source.fixAll": "explicit",
```

Comment:
Not sure if this was intentional or not. I think it is fine to leave this though. It looks like VS code changed this prop from a boolean to an enum and your editor likely migrated it automatically

https://code.visualstudio.com/updates/v1_85#_code-actions-on-save-and-auto

Reply:
Yeah, I went to back it out and it looked like it got auto migrated so I left it.